Overview.src.html

```diff
@@ -4224,7 +4224,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <p>To <dfn title=concept-BodyInit-extract>extract</dfn> a <span title=concept-body>body</span> and a
 `<code title>Content-Type</code>` <span title=concept-header-value>value</span> from
-<var>object</var>, run these steps:
+<var>object</var> with optional <var>keepalive-body-size</var>, run these steps:
```

Comment:
`keepaliveBodySize` (`Content-Type` is an exception that should maybe be modified at some point).